R/combinedscatterplot.R

```diff
@@ -19,6 +24,8 @@ CombinedScatter <- function(x = NULL,
                             scatter.colors.name = NULL,
                             scatter.colors.column = 4,
                             scatter.colors.as.categorical = TRUE,
+                            scatter.groups = NULL,
```

Comment:
I assume changes will need to be made to the Plugins code to use these new parameters?

Reply:
And that this is better than using the previous way of specifying groups which had issues?

Reply:
`scatter.groups` is actually the variable that determines how the points are separated into the small multiples. It needs to be this name because that is the same param name as used in the SmallMultiples function. It actually doesn't require a change to Plugins because it already deals with this

The code in `Scatter` sometimes uses a variable called `groups` but this is actually `scatter.colors` which I have renamed to remove confusion